src/Physics/NuElectron/EventGen/NuEPrimaryLeptonGenerator.h

```diff
@@ -32,6 +32,8 @@ public :
 
   //-- implement the EventRecordVisitorI interface
   void ProcessEventRecord(GHepRecord * event_rec) const;
+
+  virtual TVector3 EleRestFrame2Lab (GHepRecord * ev) const;
```

Comment:
Since we have to add a new method, can we change its signature to avoid pointer? 
```C++
  virtual TVector3 EleRestFrame2Lab (GHepRecord &* ev) const;
```
Also, why virtual?